parking_permits/views.py

```diff
@@ -333,6 +333,12 @@ def post(self, request, format=None):
             for permit in order.permits.all():
                 permit.status = ParkingPermitStatus.VALID
 
+                # Subscription renewed type order has always only one permit
+                if order.type == OrderType.SUBSCRIPTION_RENEWED:
+                    permit.end_time = permit.end_time + relativedelta(months=1)
```

Comment:
One question: is `end_time` guaranteed to be NOT NULL?

Reply:
I'll add a null-check here.